### ISSUE TITLE
"Raw points" option for histogram in "Display Statistics"

### ISSUE THREAD

**Issue by tedkirkpatrick** [sfu-fas/coursys#104]
The histogram in "Display Statistics" is quite useful in its percentage form.  However, some times I want to see where a given student's score falls within that histogram.  The student's score will be something like "37 / 63" and to find their location in the distribution, I have to mentally convert their score to percentage or grab a calculator.  Having an option to display the histogram in raw points would simplify that.  There are also times I want to look at the class distribution in terms of raw points.

Shifting the scale to to an arbitrary maximum complicates the code. I think it might be OK to always break  the range into 10 equal parts.  Range displays could be rounded to the nearest integer. This would require slightly more general code but not as complex as histograms for arbitrary data.  Perhaps the toughest design question would be formatting range boundaries when the maximum points is above 111, meaning the final range would be "100--111", considerably wider than the XX-XX ranges in the current plot. And what if an instructor defines a maximum of 1000 or more?  An expedient design in that case might be to display ranges in deca-points, the two most significant digits of each range boundary.

In short, I think instructors would benefit from a way to display the histogram ranges with respect to the raw scale. The code doesn't have to be nearly as general as for arbitrary data and displaying rough range cutoffs would still be useful.

There are also general-purpose histograms available (see https://plot.ly/javascript/histograms/) but they would likely be overkill both in terms of code bloat and UI complexity.
